ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -199,6 +202,41 @@ public void SetPointerEvents(BorderedCanvas view, string pointerEventsValue)
             view.SetPointerEvents(pointerEvents);
         }
 
+        /// <summary>
+        /// Set accessibility traits for the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="accessibilityTraitsValue">Can be <see cref="JArray"/> of objects or a single object. 
+        ///     String representation of the object(s) is parsed as <see cref="AccessibilityTrait"/>.</param>
+        [ReactProp("accessibilityTraits")]
+        public void SetAccessibilityTraits(BorderedCanvas view, object accessibilityTraitsValue)
+        {
+            AccessibilityTrait[] result = null;
+            if (accessibilityTraitsValue != null)
+            {
+                if (accessibilityTraitsValue is JArray asJArray)
+                {
+                    var list = new List<AccessibilityTrait>();
+                    foreach (var s in asJArray.Values<string>())
+                    {
+                        if (EnumHelpers.TryParse<AccessibilityTrait>(s, out var accessibilityTrait))
+                        {
+                            list.Add(accessibilityTrait);
+                        }
+                    }
+                    result = list.Any() ? list.ToArray() : null;
```

Comment:
I think it may be simple to have an empty array, you will avoid checking for view.AccessibilityTraits == null

Reply:
using empty array will actually complicate code since then you would need to check for both null and empty array. First, these two cases have same semantics here, so having them as two different conditions is unneeded complexity. Second, empty array takes memory, and you cannot even add to it in this model, so it's totally unused memory.

---
In reply to: [168846943](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168846943) [](ancestors = 168846943)